CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# 4.14.0
+
+- Allow use of multiple GA customDimensionIndex. See [this section](https://github.com/alphagov/govuk_frontend_toolkit/blob/master/docs/javascript.md#using-google-custom-dimensions-with-your-own-statistical-model) of the documentation for more information.
+- Configurable duration (in days) for AB Test cookie. See [this section](https://github.com/alphagov/govuk_frontend_toolkit/blob/master/docs/javascript.md#multivariate-test-framework) of the documentation for more information.
+- Add assignment of jQuery explicitly so it's consistent with other modules.
```

Comment:
I think it might have gotten lost in force-pushes, but I agree with @robinwhittleton that "Add assignment of jQuery explicitly" is an internal change, which doesn't affect consumers, so we can omit it from the changelog too 👍